apps/frontend/src/components/ViewFormsList/ViewFormsList.tsx

```diff
@@ -30,28 +30,54 @@ export default function ViewFormsList() {
   const [forms, setForms] = useState<FormData[]>([]);
   const [allForms, setAllForms] = useState<FormData[]>([]); // this is used to get all forms again after removing a filter/search term
   const [sortBy, setSortBy] = useState<SortOptions>(SortOptions.NAME);
-
   const [searchTerm, setSearchTerm] = useState<string>('');
   const navigate = useNavigate();
-
   //navigate to an user's form based on the formId
   const navigateToForm = (formId: string) => {
     navigate(`/form/${formId}`);
   };
-
   const getForms = async () => {
     const allForms = await getAllForms();
     setForms(allForms);
     setAllForms(allForms);
   };
+  useEffect(() => {
```

Comment:
I think this useEffect is duplicated here and in the useFormsListFilteringHook. 

The functionality for filtering doesn't seem to be implemented in this pr, just the UI. That's fine, but if its the case then the only change needed in this file should be to add the MultiSelectBoxes component.

If you intend to add functionality to the filtering, then the only additional change needed here would be to pass the setStatesToFilter function into the MultiSelectBoxes and then use it in there to mutate the states that we want to filter on.